AllASPFuncs.py

```diff
@@ -162,6 +165,7 @@ def local_grid(ra_center, dec_center, wcs, npoints, size = 25, spacing = 1.0, im
     return ra_grid, dec_grid
 
 
+
```

Comment:
Why the extra blank line?  One common standard is two lines between functions.  (one line between class methods).